downstream/assemblies/platform/assembly-gateway-licensing.adoc

```diff
@@ -0,0 +1,21 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-gateway-licensing"]
+= Managing {PlatformNameShort} licensing, updates and support
+
+:context: licensing-gw
+
+Ansible is an open source software project and is licensed under the GNU General Public License version 3, as described in the link:https://github.com/ansible/ansible/blob/devel/COPYING[Ansible Source Code]
```

Comment:
Need a period at the end of the sentence.

Reply:
Fixed in https://github.com/ansible/aap-docs/pull/1883